src/pages/auth/forgotpassword.css

```diff
@@ -0,0 +1,160 @@
+body {
+  overflow: hidden;
+}
+
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap");
+
+input {
+  border: 1px solid #8692a6 !important;
+  border-radius: 10px;
+}
+
+textarea:focus,
+input[type="text"]:focus,
+input[type="password"]:focus,
+input[type="datetime"]:focus,
+input[type="datetime-local"]:focus,
+input[type="date"]:focus,
+input[type="month"]:focus,
+input[type="time"]:focus,
+input[type="week"]:focus,
+input[type="number"]:focus,
+input[type="email"]:focus,
+input[type="url"]:focus,
+input[type="search"]:focus,
+input[type="tel"]:focus,
+input[type="color"]:focus,
+input[type="checkbox"]:checked,
+.uneditable-input:focus {
+  color: #696f79;
+  border-color: rgba(239, 200, 26, 0.8) !important;
+  box-shadow: 0 1px 1px rgba(239, 200, 26, 0.1) inset,
+    0 0 6px rgba(239, 200, 26, 0.9) !important;
+  outline: 0 none;
+}
+
+input[type="checkbox"]:focus {
+  box-shadow: none !important;
+  outline: 0 none;
+}
+
+input[type="checkbox"]:checked {
+  background: #efc81a;
+}
+
+input::placeholder {
+  font-family: "Inter";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  display: flex;
+  align-items: center;
+  color: #8692a6 !important;
+}
+
+input.form-control {
+  width: 100%;
+  height: 2.825rem !important;
+  background: #ffffff !important;
+}
+
+input.form-control:active {
+  background: #ffffff;
+}
+
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus,
+input:-webkit-autofill:active {
+  -webkit-box-shadow: 0 0 0 30px white inset !important;
+}
+
+label.inputLabelText {
+  font-family: "Inter";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 19px;
+  display: flex;
+  align-items: center;
+  color: #696f79 !important;
+}
+
+.btn-warning {
+  background-color: #efc81a !important;
+  height: 2.625rem;
+}
+
+.title-text,
+.signup-text {
+  color: #efc81a;
+}
+
+.subtitle-text,
+.forgot-password-text,
+.dont-have-an-account-text,
+.terms-and-conditions-text {
+  color: #8692a6;
+}
+
+.title-text,
+.subtitle-text,
+.forgot-password-text,
+.signup-text,
```

Comment:
@abdussalim Please delete this. Do not need this in forgotpassword.css. Thank you.